Client/Frontend/Autofill/CreditCard/CreditCardSettingsView/CreditCardSettingsViewController.swift

```diff
@@ -0,0 +1,136 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import UIKit
+import Shared
+import SwiftUI
+
+class CreditCardSettingsViewController: UIViewController, ThemeApplicable {
+    var themeObserver: NSObjectProtocol?
+    var theme: Theme
+    var viewModel: CreditCardSettingsViewModel
+    var state: CreditCardSettingsState = .empty
+    var startingConfig: CreditCardSettingsStartingConfig?
+
+    // MARK: Views
+    var creditCardEmptyView: UIHostingController<CreditCardSettingsEmptyView>
+    var creditCardAddEditView: UIHostingController<CreditCardEditView>
+    var creditCardTableViewController: CreditCardTableViewController
+
+    // MARK: Initializers
+    init(theme: Theme,
+         creditCardViewModel: CreditCardSettingsViewModel,
+         startingConfig: CreditCardSettingsStartingConfig?) {
+        self.theme = theme
+        self.startingConfig = startingConfig
+        self.viewModel = creditCardViewModel
+        self.creditCardEmptyView = UIHostingController(rootView: CreditCardSettingsEmptyView())
+        self.creditCardAddEditView =
+        UIHostingController(rootView: CreditCardEditView(
+            viewModel: viewModel.addEditViewModel,
+            removeButtonColor: Color(theme.colors.textWarning),
+            borderColor: Color(theme.colors.borderPrimary)))
+        self.creditCardTableViewController = CreditCardTableViewController(theme: theme, viewModel: viewModel.creditCardTableViewModel)
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("not implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        applyTheme(theme: theme)
+        viewSetup()
+    }
+
+    func viewSetup() {
+        guard let emptyCreditCardView = creditCardEmptyView.view else { return }
+        guard let addEditCreditCardView = creditCardAddEditView.view else { return }
+        guard let creditCardTableView = creditCardTableViewController.view else { return }
```

Comment:
```suggestion
        guard let emptyCreditCardView = creditCardEmptyView.view,
            let addEditCreditCardView = creditCardAddEditView.view,
            let creditCardTableView = creditCardTableViewController.view else { return }
```
We could merge the 3 guard in 1

Reply:
good cleanup!!